athena/solver.py

```diff
@@ -64,6 +65,8 @@ def initialize_devices(visible_gpu_idx=None):
             tf.config.experimental.set_memory_growth(gpu, True)
         # means we're running in GPU mode
         if len(gpus) != 0:
+            if len(visible_gpu_idx) == 0:
+                visible_gpu_idx.append(0)
```

Comment:
what will happen if the user just doesn't want to use gpu?

Reply:
Please add some comments for user w.r.t. CUDA_VISIBLE_DEVIDES, then we shall merge this pr